fsspec/utils.py

```diff
@@ -500,7 +552,7 @@ def merge_offset_ranges(paths, starts, ends, max_gap=0, max_block=None, sort=Tru
     if not isinstance(starts, list):
         starts = [starts] * len(paths)
     if not isinstance(ends, list):
-        ends = [starts] * len(paths)
+        ends = [ends] * len(paths)
     if len(starts) != len(paths) or len(ends) != len(paths):
         raise ValueError
```

Comment:
This seems like a bug, shouldn't we actually instatiate the error, and ideally print an error message about why it's thrown?

Reply:
A message would be a reasonable thing to add here, but it's not necessary to instantiate if there is no message - python will do it automatically.
```
>>> raise ValueError
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError
```